users/management/commands/retire_users.py

```diff
@@ -0,0 +1,124 @@
+"""
+Retire user(s) from MIT xPRO
+"""
+from argparse import RawTextHelpFormatter
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+
+from social_django.models import UserSocialAuth
+
+from user_util import user_util
+
+User = get_user_model()
+
+RETIRED_USER_SALTS = ["mitxpro-retired-email"]
+RETIRED_EMAIL_FMT = "retired_email_{}@retired.mit.edu"
```

Comment:
One last nit: change the domain to `"@retired.{urlparse(settings.SITE_BASE_URL).netloc}"` so the emails live under the app domain